test/unit/io/alignment_file/alignment_file_format_test_template.hpp

```diff
@@ -14,7 +14,7 @@
 #include <seqan3/io/alignment_file/output_format_concept.hpp>
 #include <seqan3/io/alignment_file/input.hpp>
 #include <seqan3/io/alignment_file/output.hpp>
-#include <seqan3/range/view/convert.hpp>
+#include <seqan3/range/views/convert.hpp>
```

Comment:
```suggestion
#include <seqan3/range/views/take.hpp>
```

Reply:
view::convert is being used in this file